.github/workflows/build.yml

```
@@ -140,9 +140,16 @@ jobs:
               `;
             }, '' );
             const commentBody = `${ TEST_COMMENT_INDICATOR }
-            Are you an Automattician? You can now test your Pull Request on WordPress.com. On your sandbox, run
+            Are you an Automattician? Please test your changes on WordPress.com.
+
+            - To test on WoA, install [the Beta plugin](https://jetpack.com/download-jetpack-beta/), go to Jetpack > Jetpack Beta > Jetpack and enable the \`${ BRANCH_NAME }\` branch.
+            - To test on Simple, run the following command on your sandbox:
             ${ commands }
-            to get started. More details: p9dueE-5Nn-p2`;
+
```

```suggestion
            Note: Because the command applies the entire Jetpack `trunk`, you may end up with a lot of changed files in your sandbox. Make sure to clean all of them up once you're done with your testing.

```

Hopefully this applies cleanly after the last command.

---

I think this would be covered with the above:
https://github.com/Automattic/jetpack/pull/31266/files#r1223229356

---

76cd497d2317007b86fec2cb280b8ab6affb3348 should cover this.